W-18587853-MMP-RFTautoscalingParameter-45-dm (#3040)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/_partials/mmp-deploy-to-rtf.adoc

| `disableAmLogForwarding` | Disables the application-level log forwarding to Anypoint Monitoring. By default, it is set to `false`.

> | `autoscaling` | Use this object to provide CPU Based Horizontal Autoscaling configuration on deployment and redeployment operations. If autoscaling is disabled and the fields `minReplicas` and `maxReplicas` are provided, they must match the value of the `target.replicas` field. See xref:cloudhub-2::ch2-configure-horizontal-autoscaling.adoc[Enable Trace Data Collection]. 

Reviewer: ```suggestion
| `autoscaling` | Provides CPU based horizontal autoscaling configuration for deployment and redeployment operations. If autoscaling is disabled and the fields `minReplicas` and `maxReplicas` are provided, they must match the value of the `target.replicas` field. See xref:cloudhub-2::ch2-configure-horizontal-autoscaling.adoc[Enable Trace Data Collection]. 
```

Reviewer: Is the acronym missing in "CPU Based Horizontal Autoscaling"? Why the capital letters?
